### PR TITLE
chore: `"exports"` to include the `debug` entry

### PR DESCRIPTION
**Pull request by maraisr**
The package is configured with `type: module` but trying to import the `/debug` entry throws this error:

```
Package subpath './debug' is not defined by "exports"
```

>> lets add it.

Wasn't sure if we need a `debug.d.ts`, not sure how Typescript will behave with it missing, but it feels right.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +37,17 @@
...
+  "exports": {
+    ".": {
+      "types":"./src/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./debug": {
+      "types":"./src/debug.d.ts",
+      "import": "./dist/debug.js"
+    },
+    "./package.json": "./package.json"
```
Comment (TomerAberbach) [inline package.json:49]: Curious why this is needed? Do people typically need to import `package.json` for some reason?

**Review by TomerAberbach — COMMENTED** [review]
Thanks! LG, but just have a question about one of the changes.

File: package.json (hunk 1 above)
Comment (maraisr) [reply package.json:49]: not typically people, but machines do. and without this listing, node fails to tell the bundler about the export map.

here's a few examples: 

svelte: https://github.com/sveltejs/svelte/blob/master/package.json#L24
polka: https://github.com/lukeed/polka/blob/next/packages/polka/package.json#L15
vite: https://github.com/vitejs/vite/blob/main/packages/vite/package.json#L31

but think maybe @lukeed has a better answer for you.

**Comment by TomerAberbach** [comment]
Released!

File: package.json (hunk 1 above)
Comment (lukeed) [reply package.json:49]: it's just a good idea to have in general. things like rollup plugins may try to access a `package.json` file directly, so this just wards off the inevitable request to expose it

**Pull request merged**